### PR TITLE
Update irmaAuthentication.php

### PR DESCRIPTION
**Pull request by wvbattum**

### CODE REVIEW

File: views/irmaAuthentication.php [hunk 1]
```diff
@@ -1,13 +1,77 @@
...
+      .catch(error => {
+        if (error === 'Aborted') {
+          console.log('We closed it ourselves, so no problem 😅');
+          return;
+        }
+        console.error("Couldn't do what you asked 😢", error);
+      })
+      .finally(() => irmaPopup = irma.newPopup(options));
+    };
+/*
```
Comment (ghost) [inline views/irmaAuthentication.php]: you left in a big amount of commented code...

File: views/irmaAuthentication.php [hunk 2]
```diff
@@ -1,13 +1,77 @@
...
+      // Front-end options
+      language:  'en',
+      translations: {
+        header:  'Sign the agreement with <i class="irma-web-logo">IRMA</i>',
+        loading: 'Just one second please!'
+      },
+
+      session: {
+        // IRMA server:
+        url: 'http://localhost:8088',
```
Comment (ghost) [inline views/irmaAuthentication.php]: should we make this configurable?

Comment (wvbattum) [reply views/irmaAuthentication.php]: Just for now, because I run the irma server locally

**Comment by ghost** [comment]
What is the status of this PR? Does it work? Do we have a build script for building the irma.js file? Currently it is a "blob"...

**Comment by wvbattum** [comment]
PR for IRMA

**Comment by ghost** [comment]
Fixed. See https://github.com/eduvpn/vpn-user-portal/commit/a6ddb20b30001e772af9898c726f5976d182212b

**Pull request closed without merging**